bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownAlignmentBehavior.java

```diff
@@ -43,14 +55,73 @@ public String cssClassName() {
         }
     }
 
-    private final IModel<Alignment> alignment;
+    /**
+     * Alignment for the dropdown of a DropDownButton
+     */
+    public static class Alignment implements ICssClassNameProvider {
+        private static final String CLASS_RIGHT = "end";
+        private static final String CLASS_LEFT = "start";
+        public static final Alignment RIGHT = new Alignment(CLASS_RIGHT);
```

Comment:
Do we need `LEFT` for completeness ?

Reply:
Actually `RIGHT` was added for backward compatibility :)
`Alignment.right()` and `Alignment.left()` can be used :)